src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -42,12 +53,9 @@ public abstract class NodeModel : ModelBase, IRenderPackageSource<NodeModel>, ID
         private string name;
         private ElementState state;
         private string toolTipText = string.Empty;
+        private HashSet<Info> infos = new HashSet<Info>();
         private string description;
         private string persistentWarning = string.Empty;
```

Comment:
do we still need `persistentWarning` as a separate property ?
Or can we look it/them up in the new `infos` hashset ?

Reply:
I guess previously, warning bubble always fallback to this so persistent warnings can be isolated from execution warningsos for display